Sources/Subs-Db-mysql.php

```diff
@@ -7,7 +7,7 @@
  *
  * @package SMF
  * @author Simple Machines http://www.simplemachines.org
- * @copyright 2017 Simple Machines and individual contributors
+ * @copyright 2012 Simple Machines
```

Comment:
Please ensure that you update the copyright information in all files: https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/Sources/Subs-Db-mysqli.php#L10
:)